labelbox/data/annotation_types/label.py

```diff
@@ -130,12 +131,13 @@ def assign_feature_schema_ids(
             self, ontology_builder: ontology.OntologyBuilder) -> "Label":
         """
         Adds schema ids to all FeatureSchema objects in the Labels.
-        This is necessary for MAL.
 
         Args:
             ontology_builder: The ontology that matches the feature names assigned to objects in this dataset
         Returns:
             Label. useful for chaining these modifying functions
+
+        Warning: You can now import annotations using names directly without having to lookup schema_ids
```

Comment:
Change this to `Note: You can now import annotations using names directly without having to lookup schema_ids`, and remove `import warning`?